crm_lead_to_event_registration/models/crm_lead.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    event_registration_id = fields.Many2one(
+        "event.registration",
+        "Event registration",
+        readonly=True,
+        help="Registration generated by this lead/opportunity.")
+
+    @api.multi
+    def action_generate_event_registration(self, event):
+        """Generate an event registration."""
+        er = self.env["event.registration"]
+        for s in self:
+            s.event_registration_id = er.create({
+                "event_id": event.id,
+                "partner_id": s.partner_id.id,
+                "nb_register": 1,
+                "name": s.contact_name,
```

Comment:
But this behavior has no sense. I think that implement the fallback and raise a ValidationError helps user and increase UX. And It's very easy to implement BTW